src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -768,8 +768,7 @@ func (s *service) FetchBlocksRaw(tctx thrift.Context, req *rpc.FetchBlocksRawReq
 	// Preallocate starts to maximum size since at least one element will likely
 	// be fetching most blocks for peer bootstrapping
 	ropts := nsMetadata.Options().RetentionOptions()
-	blockStarts := make([]time.Time, 0,
-		(ropts.RetentionPeriod()+ropts.FutureRetentionPeriod())/ropts.BlockSize())
+	blockStarts := make([]time.Time, 0, ropts.RetentionPeriod()/ropts.BlockSize())
```

Comment:
I was thinking that since we won't be supporting future writes for now, we should remove this for now. Although I support `FutureRetentionPeriod` would be 0 and so this should be a no op. I'll keep this in.